vehicle/mkvi/software/sensing/sensing.c

```diff
@@ -0,0 +1,69 @@
+/*
+Header:
+    Code for the Sensing Controller Board located in the nostril motherboard and
+the rear sensing box Author:
+    @author Richard Li
+    @date   03/25/2023
+*/
+
+/*----- Includes -----*/
+#include "sensing.h"
+#include "can_api.h"
+#include "log_uart.h"
```

Comment:
Is this import used anywhere?

Reply:
This is a relic from old code and should be removed.